src/main/java/org/opensearch/ad/ml/EntityModel.java

```diff
@@ -32,24 +32,14 @@
 
 import org.opensearch.ad.model.Entity;
 
-import com.amazon.randomcutforest.RandomCutForest;
-import com.amazon.randomcutforest.parkservices.threshold.ThresholdedRandomCutForest;
+import com.amazon.randomcutforest.parkservices.ThresholdedRandomCutForest;
 
 public class EntityModel {
     private Entity entity;
     // TODO: sample should record timestamp
     private Queue<double[]> samples;
-    private RandomCutForest rcf;
-    private ThresholdingModel threshold;
     private ThresholdedRandomCutForest trcf;
 
-    public EntityModel(Entity entity, Queue<double[]> samples, RandomCutForest rcf, ThresholdingModel threshold) {
-        this.entity = entity;
-        this.samples = samples;
-        this.rcf = rcf;
-        this.threshold = threshold;
-    }
-
     /**
      * Constructor with ERCF.
```

Comment:
Change to tRCF? Not urgent/not important right now.

Reply:
changed.